Микросетки на флексах для всех страниц

alexxxey2308 opened the pull request:
---
:mortar_board: [Микросетки на флексах для всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2110521/tasks/11)

:boom: https://htmlacademy-htmlcss.github.io/2110521-sedona-35/9/

---
:mortar_board: [Постройте мелкие сетки на гридах для подходящих компонентов](https://up.htmlacademy.ru/htmlcss/35/user/2110521/tasks/12)

alexxxey2308 committed:
Микросетки на флексах для всех страниц

alexxxey2308 committed:
доделал пагинацию

saimon395 commented on index.html:
```diff
@@ -62,8 +62,8 @@ <h2 class="visually-hidden" hidden>Преимущества</h2>
                   <h3 class="advantages-item-title">Настоящий городок</h3>
-                  <svg class="advantages-item-svg" width="60" height="2" viewBox="0 0 60 2" fill="none"
-                    xmlns="http://www.w3.org/2000/svg">
+                  <!-- <svg class="advantages-item-svg" width="60" height="2" viewBox="0 0 60 2" fill="none"
+                    xmlns="http://www.w3.org/2000/svg"> -->
                     <rect width="60" height="2" fill="white" fill-opacity="0.3" />
                   </svg>
```
если удаляешь свгшку , удаляй полностью, не оставляй закрывающие тэги

saimon395 commented on catalog.html:
```diff
@@ -131,25 +131,31 @@ <h2 class="visually-hidden" hidden>Список жилья с фильтрами
-              </li>
-            </ul>
-            <span class="product-card-rating">Рейтинг: 8,5</span>
+            <div class="product-card-wrapper">
+              <span class="product-card-hotel">Гостиница</span>
+              <span class="product-card-price">От 4000 ₽</span>
+            </div>
+            <div class="product-card-button-wrapper">
+              <a class="product-card-button-brown" href="#">подробнее</a>
+              <a class="product-card-button-favourite" href="#">в избранное</a>
```
а зачем здесь врапер ? а если всё таки он тут нужен, то почему к остальным элементам списка не добавил ?

alexxxey2308 replied:
все лишнее удалил.

alexxxey2308 committed:
убрал свг и враппер из карточек товара

saimon395 commented on index.html:
```diff
@@ -62,8 +62,6 @@ <h2 class="visually-hidden" hidden>Преимущества</h2>
                   <h3 class="advantages-item-title">Настоящий городок</h3>
-                  <svg class="advantages-item-svg" width="60" height="2" viewBox="0 0 60 2" fill="none"
-                    xmlns="http://www.w3.org/2000/svg">
                     <rect width="60" height="2" fill="white" fill-opacity="0.3" />
                   </svg>
```
rect и закрывающий svg удали

saimon395 replied:
перед отправкой на проверку разметку через валидатор прогони https://validator.w3.org/nu/#textarea

alexxxey2308 committed:
карточки товара

сделал через грид

alexxxey2308 committed:
добавил justify-self к цене

alexxxey2308 committed:
удалил rect

alexxxey2308 committed:
прогнал через валидатор, исправил то, что мог

переработал блок с преимуществами.
остались ошибки в css файле первая строка, где подключение шрифтов. не знаю как исправить это

alexxxey2308 committed:
добавил отступы футер

saimon395 commented on styles/styles.css:
```diff
@@ -389,17 +474,47 @@ body {
   font-size: 30px;
   line-height: 36px;
+  margin: 0;
+  padding: 0;
+  margin-left: 70px;
+  margin-right: 200px;
```
ты вот с такими марджинами поаккуратнее будь, сетка крайне не стабильная получается, чуть контент поменяешь и всё поедет, критерий про переполнение соответственно тоже не пройдёт, поэтому переделай например на гриды, если сетку правильно задать, марджины не понадобятся, в остальных местах где так делаешь тоже поправь

alexxxey2308 committed:
исправил сетку
